examples/edge-functions/supabase/functions/select-from-table-with-auth-rls/index.ts

```diff
@@ -0,0 +1,33 @@
+// Follow this setup guide to integrate the Deno language server with your editor:
+// https://deno.land/manual/getting_started/setup_your_environment
+// This enables autocomplete, go to definition, etc.
+
+import { serve } from 'https://deno.land/std@0.131.0/http/server.ts'
+import { createClient } from 'https://esm.sh/@supabase/supabase-js@^1.33.2'
+
+console.log(`Function "select-from-table-with-auth-rls" up and running!`)
+
+serve(async (req: Request) => {
+  const supabase = createClient(
```

Comment:
perhaps we should move the supabase client out of the function then use setAuth?

```js
const supabase = createClient(Deno.env.get('SUPABASE_URL'), Deno.env.get('SUPABASE_ANON_KEY'))

serve(async (req: Request) => {
  supabase.setAuth(req.headers.get('Authorization'))
  // ...
})
```

the difference isn't huge, but just feels cleaner